src/resources/platform-adaptations.md

```diff
@@ -560,6 +565,180 @@ This section includes preliminary recommendations on how to adapt
+
+### Top app bar and navigation bar
+
+Since Android 12, the default UI for top app 
+bars follow the design guidelines defined in [Material 3][mat-appbar]. 
+On iOS, an equivalent component called "Navigation Bars" 
+is defined in [Apple’s Human Interface Guidelines][hig-appbar] (HIG). 
+
+<div class="container">
+  <div class="row">
+    <div class="col-sm text-center">
+      <figure class="figure">
+        <img src="/assets/images/docs/platform-adaptations/mat-appbar.png" 
+        class="figure-img img-fluid rounded" alt=" Top App Bar in Material 3 " />
+        <figcaption class="figure-caption">
+          Top App Bar in Material 3 
+        </figcaption>
+      </figure>
+    </div>
+    <div class="col-sm">
+      <figure class="figure text-center">
+        <img src="/assets/images/docs/platform-adaptations/hig-appbar.png" 
+        class="figure-img img-fluid rounded" alt="Navigation Bar in Human Interface Guidelines" />
+        <figcaption class="figure-caption">
+          Navigation Bar in Human Interface Guidelines
+        </figcaption>
+      </figure>
+    </div>
+  </div>
+</div>
+
+Certain properties of app bars in Flutter apps should be adapted, 
+like system icons and page transitions. 
+These are already automatically adapted when using 
+the Material `AppBar` and `SliverAppBar` widgets. 
+You can also further customize the properties of these widgets to better 
+match iOS platform styles, as shown below. 
+
+```dart
+// Map the text theme to iOS styles
+TextTheme cupertinoTextTheme = TextTheme(
+    headlineMedium: CupertinoThemeData()
+        .textTheme
+        .navLargeTitleTextStyle
+         // fixes a small bug with spacing
+        .copyWith(letterSpacing: -1.5),
+    titleLarge: CupertinoThemeData().textTheme.navTitleTextStyle)
+...
+
+// Use iOS text theme on iOS devices
+ThemeData(
+      textTheme: Platform.isIOS ? cupertinoTextTheme : null,
+      ...
+)
+...
+
+// Modify AppBar properties
+AppBar(
+        surfaceTintColor: Platform.isIOS ? Colors.transparent : null,
+        shadowColor: Platform.isIOS ? CupertinoColors.darkBackgroundGray : null,
+        scrolledUnderElevation: Platform.isIOS ? .1 : null,
+        toolbarHeight: Platform.isIOS ? 44 : null,
+        ...
+      ),
+
+```
+
+But, because app bars are displayed alongside 
+other content in your page, it's only recommended to adapt the styling 
+so long as its cohesive with the rest of your application. You can see 
+additional code samples and a further explanation in [the Github discussion on app bar adaptations][appbar-post]. 
+
+### Bottom navigation bars
+
+Since Android 12, the default UI for bottom navigation 
+bars follow the design guidelines defined in [Material 3][mat-navbar]. 
+On iOS, an equivalent component called "Tab Bars" 
```

Comment:
Same as above. Link on "Tab Bars", not on the "Apple's Human...", but keep the text there.